integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -189,10 +189,52 @@ case class CreateTable(cm: TableModel) extends RunnableCommand {
       // Need to fill partitioner class when we support partition
       val tablePath = catalog.createTableFromThrift(tableInfo, dbName, tbName, null)(sqlContext)
       try {
-        sqlContext.sql(
-          s"""CREATE TABLE $dbName.$tbName USING carbondata""" +
-          s""" OPTIONS (tableName "$dbName.$tbName", tablePath "$tablePath") """)
-          .collect
+        val useCompatibleSchema = sqlContext.sparkContext.conf
```

Comment:
@cenyuhai changes in this class is not required till now we are only supporting hive with spark 2.1 and hadoop 2.7